.github/workflows/pr-review.yml

```
@@ -0,0 +1,108 @@
+    types: [opened, synchronize, reopened]
+    branches: [main, develop]
+
+jobs:
+  # ------------------------------------------------------------------
+  # Step 1: Detect which domains were touched so we only invoke the
+  # relevant personas. McCore has two: extensibility and testing.
+  # ------------------------------------------------------------------
+  detect-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      needs_api:  ${{ steps.detect.outputs.api }}
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)